### ISSUE TITLE
Is it possible to regenotype a VCF using giraffe?

### ISSUE THREAD

**Issue by rickymagner** [vgteam/vg#4330]
Hi, I posted this question on Biostars [here](https://www.biostars.org/p/9590235/) a few months ago but never received a response. 

In short, is it possible to both:
- Use giraffe to create a GAM from reads against a VCF-augmented graph
- Regenotype the same VCF using the read evidence from `vg call -v`? 

The README says that the indices/files created in the giraffe workflow cannot be used when running `vg call -v` . Is it correct to conclude that there is no possible way to use both of these tools together?

Thanks

**Comment by jeizenga** [comment]
Yes, it is possible, but we don't currently have a workflow for it in `vg autoindex` (previously noted here: https://github.com/vgteam/vg/issues/3974), so you would need to use the manual indexing pipeline:`vg construct -a`, `vg gbwt -v`, `vg gbwt --gbz-format`, `vg index -j`, `vg minimizer -d`.

**Comment by faithokamoto** [comment]
Note for future: people also want to be able to do this for unphased VCFs
